@xen-orchestra/web/src/components/vm/VmGeneralInfo.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <UiCard class="vm-general-info">
+    <UiTitle>
+      {{ $t('general-information') }}
+    </UiTitle>
+    <VtsQuickInfoRow :label="$t('name')" :value="vm.name_label" />
+    <VtsQuickInfoRow :label="$t('id')" :value="vm.id" />
+    <VtsQuickInfoRow :label="$t('description')" :value="vm.name_description" />
+    <VtsQuickInfoRow :label="$t('tags')">
+      <template v-if="vm.tags.length > 0" #value>
+        <UiTagsList>
+          <UiTag v-for="tag in vm.tags" :key="tag" accent="info" variant="secondary">{{ tag }}</UiTag>
+        </UiTagsList>
+      </template>
+    </VtsQuickInfoRow>
+    <VtsQuickInfoRow :label="$t('os-name')" :value="vm?.os_version?.name" />
+    <VtsQuickInfoRow :label="$t('os-kernel')" :value="vm?.os_version?.uname" />
```

Comment:
Since the `vm` prop is required, it can't be `undefined`, and you don't need to check for that:
```suggestion
    <VtsQuickInfoRow :label="$t('os-name')" :value="vm.os_version?.name" />
    <VtsQuickInfoRow :label="$t('os-kernel')" :value="vm.os_version?.uname" />
```